src/main/java/org/opentripplanner/routing/core/Money.java

```diff
@@ -56,10 +56,6 @@ public void setCurrency(Currency currency) {
         this.currency = new WrappedCurrency(currency);
     }
     
-    public void setCurrency(WrappedCurrency currency) {
```

Comment:
This seems like a good change, but why was it made as part of this PR?

Reply:
I removed this unused method as Jackson was getting confused as to which constructor should be used. In the end I couldn't use it anyway so will restore it to its previous state.